manifests/config.pp

```diff
@@ -67,9 +67,11 @@
         }
       }
       'systemd' : {
-        include 'systemd'
-        systemd::unit_file {'prometheus.service':
-          content => template('prometheus/prometheus.systemd.erb'),
+        if $prometheus::server::manage_systemd_unit {
+          include 'systemd'
+          systemd::unit_file {'prometheus.service':
```

Comment:
instead of defining the `systemd::unit_file` resoure here, should we only set `$ensure = 'present` and pass that to the resource? And if `$prometheus::server::manage_systemd_unit` isn't true, set `$ensure = 'absent'.

Reply:
that's what i said earlier:

> i wonder if we shouldn't enforce that in the class instead of wrapping the whole systemd thing inside an `if`. but then it would destroy local customizations people might have as well, so maybe it's better to let the users handle the transition since we don't support debian packages officially anyways right now...

i don't mind either way...